<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TpCameraClass</class>
 <widget class="QMainWindow" name="TpCameraClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TpCamera</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Allumer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Eteindre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Bas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Droite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Haut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Zoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Déplacement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remise position zéro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Balayage</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TpCamera.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonAllumer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonEteindre()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonDeZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonStopZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonGauche()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonBas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonDroite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonHaut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonStopMouvement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonRemiseZero()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked(bool)</signal>
   <receiver>TpCameraClass</receiver>
   <slot>onButtonBalayage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonAllumer()</slot>
  <slot>onButtonEteindre()</slot>
  <slot>onButtonZoom()</slot>
  <slot>onButtonDeZoom()</slot>
  <slot>onButtonStopZoom()</slot>
  <slot>onButtonGauche()</slot>
  <slot>onButtonDroite()</slot>
  <slot>onButtonBas()</slot>
  <slot>onButtonHaut()</slot>
  <slot>onButtonStopMouvement()</slot>
  <slot>onButtonRemiseZero()</slot>
  <slot>onButtonBalayage()</slot>
 </slots>
</ui>
